client/auth/controller.js

```diff
@@ -20,16 +22,27 @@ import userFactory from 'lib/user';
 import Main from 'components/main';
 import PulsingDot from 'components/pulsing-dot';
 
-module.exports = {
+export default {
 
-	// Login screen used by the desktop application
-	login: function() {
-		if ( OAuthToken.getToken() ) {
-			page( '/' );
-		} else {
-			ReactDom.render(
-				React.createElement( LoginComponent, {} ),
-				document.getElementById( 'primary' )
+	login: function( context ) {
+		if ( config.isEnabled( 'oauth' ) ) {
+			// hack to keep the blue background for Desktop login
+			document.querySelector( '.layout' ).style.backgroundColor = '#0087be';
```

Comment:
Hi, yeah the desktop apps login design is made to mimic the mobile apps. And yes, @josemarques is right about it doubling as a splash screen. We also do a few other things, like show pin app nudges:

![first-run-mac](https://cloud.githubusercontent.com/assets/1204802/24511771/f5b90e96-156c-11e7-87b1-ef395028e422.png)